graphistry/text_utils.py

```diff
@@ -285,5 +291,9 @@ def load_search_instance(self, savepath):
         from joblib import load  # type: ignore   # need to make this onnx or similar
 
         cls = load(savepath)
-        cls.build_index()
+        try:
+            cls.build_index()
+        except Exception as e:
+            logger.exception(e)
+            logger.warn("Could not build index, run g.build_index() to build it later")
```

Comment:
`import warnings; ...` ?

Reply:
why aren't we raising?

Reply:
Can we detect the expected-to-fail scenario ahead of time, skip building in that scenario, and still raise for unexpected errors? This seems to catch too many exn cases otherwise and thus is known-wrong..